### DISCUSSION TITLE
Workspace Notifications Configuration via API?

### DISCUSSION THREAD

**Discussion by dominon12** [airbytehq/airbyte#44852] [Platform API Questions] [answered]
Hi everyone,

I couldn't find specific details in the API documentation, so I wanted to ask here: Is there an option to configure notifications via the API? Specifically, I'm looking to disable certain types of email notifications and enable webhooks. Any guidance or pointers would be appreciated!

**Comment by nataliekwong** [comment, marked as answer]
There is not currently a way to configure the notification settings through the API. Unfortunately this can only be done through the UI today. I appreciate the suggestion and will add it to our feature roadmap!

**Reply by dominon12** [reply]
Thanks for your answer. That's very sad that it's not yet supported. Is there an option to subscribe on the feature status?

**Reply by nataliekwong** [reply]
I created an issue for tracking here: https://github.com/airbytehq/airbyte/issues/44889. Feel free to subscribe to it or +1 to get it prioritized on our roadmap!
